jsdojo/slabtext.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+    "use strict";
+    $( document ).ready(function() {
+        kintone.events.on('app.record.detail.show', function(event) {
+            var textField = kintone.app.record.getFieldElement('Text');
+            console.log(textField);
+            var text = event.record.Text.value;
+            var slab0 = "<span class='slabtext'>",
+                span0 = "</span>",
+                txt0 = [text];    
+            $(textField).html(slab0 + txt0.join(span0 + slab0) + span0).slabText();
```

Comment:
Using the form input as the argument of `$.html()` can be dangerous because users can execute codes.
https://stackoverflow.com/questions/49396862/jquery-html-method-a-security-risk-like-innerhtml

Consider escaping the form input before passing it to `$.html()`. Please refer to the following about how to sanitize strings.
https://github.com/austincqdo/jsdojo1/pull/10#discussion_r310980078